zubhub_frontend/zubhub/src/assets/js/styles/views/project_details/projectDetailsStyles.js

```diff
@@ -105,16 +105,29 @@ const styles = theme => ({
       height: '3.5em',
       width: '100%',
       flexDirection: 'row',
-      justifyContent: 'flex-start',
+      justifyContent: 'left',
     },
   },
   actionBoxButtonStyle: {
```

Comment:
Why do we need actionBoxButton changes as part of this PR? Can you attach a screenshot?